presets/ras-defaults.json

```diff
@@ -106,7 +106,7 @@
         {
             "status": "draft",
             "title": "Inline Newsletter Signup (secondary)",
-            "content": "<!-- wp:paragraph -->\n<p>Signup for our free newsletter to receive the latest news<\/p>\n<!-- \/wp:paragraph -->\n\n<!-- wp:newspack-newsletters\/subscribe \/-->\n\n<!-- wp:separator -->\n<hr class=\"wp-block-separator has-alpha-channel-opacity\"\/>\n<!-- \/wp:separator -->",
+            "content": "<!-- wp:group {\"className\":\"is-style-border\",\"layout\":{\"type\":\"constrained\"}} -->\n<div class=\"wp-block-group is-style-border\"><!-- wp:paragraph -->\n<p>Signup for our free newsletter to receive the latest news<\/p>\n<!-- \/wp:paragraph -->\n\n<!-- wp:newspack-newsletters\/subscribe \/--><\/div>\n<!-- \/wp:group -->",
```

Comment:
Nit: Prompts have a `hide_border` attribute which is `false` by default, but `1` in this prompt's preset config. If you remove it or set it to `false` you don't have to include the wrapping Group block or its border style, and Campaigns will apply a standard border style to the prompt container div that matches all prompts.

Reply:
Good call. I've updated the PR.

I was falling the instructions Tracy added to the Asana task. Do you think there could be a reason for her to prefer it that way? (I'm asking)

Reply:
Checked with Tracy and she's okay with using the global style instead of the Group style.